week-7/Homework/mandatory/1-debugging-practice/script.js

```diff
@@ -29,16 +29,22 @@ const check = document.getElementById("check");
 //via Book function and start render function
 function submit() {
   if (
-    title.value == null ||
-    title.value == "" ||
-    pages.value == null ||
-    pages.value == ""
+    title.value === null ||
+    author.value === "" ||// I changed title to author  5
```

Comment:
It would be better to add the pair of conditions (`== null` and `== ""`) for author, rather than replacing one of `title`s, now they're not consistent with `pages`